cli/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
```

Comment:
I don't like this change, I believe that the year should change when file is edited in next year, not globally (I might be wrong)